WordPress/Classes/ViewRelated/Reader/ReaderPostBlockingController.swift

```diff
@@ -0,0 +1,143 @@
+import Foundation
+
+protocol ReaderSiteBlockingControllerDelegate: AnyObject {
+    func readerSiteBlockingController(_ controller: ReaderPostBlockingController, willBeginBlockingSiteOfPost post: ReaderPost)
+    func readerSiteBlockingController(_ controller: ReaderPostBlockingController, didBlockSiteOfPost post: ReaderPost, result: Result<Void, Error>)
+    func readerSiteBlockingController(_ controller: ReaderPostBlockingController, willBeginBlockingPostAuthor post: ReaderPost)
+    func readerSiteBlockingController(_ controller: ReaderPostBlockingController, didFinishBlockingPostAuthor post: ReaderPost, result: Result<Void, Error>)
+}
+
+extension ReaderSiteBlockingControllerDelegate {
+    func readerSiteBlockingController(_ controller: ReaderPostBlockingController, willBeginBlockingSiteOfPost post: ReaderPost) {}
+    func readerSiteBlockingController(_ controller: ReaderPostBlockingController, didBlockSiteOfPost post: ReaderPost, result: Result<Void, Error>) {}
+    func readerSiteBlockingController(_ controller: ReaderPostBlockingController, willBeginBlockingPostAuthor post: ReaderPost) {}
+    func readerSiteBlockingController(_ controller: ReaderPostBlockingController, didFinishBlockingPostAuthor post: ReaderPost, result: Result<Void, Error>) {}
+}
+
+final class ReaderPostBlockingController {
```

Comment:
@alpavanoglu This is not really a new class, there was an existing filed named `ReaderSiteBlockingController.swift` which I renamed to `ReaderPostBlockingController.swift`. And somehow thought this is a brand new file.

That being said, the new methods added are:
- handleUserBlockingWillBegin
- handleUserBlockingDidFinish